cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -295,6 +295,11 @@ func NewManager(
 	console input.Console,
 	commandRunner exec.CommandRunner,
 ) (*Manager, error) {
+	// Use silent console when console is not provided
```

Comment:
Are you solving the problem of the container app output in `deploy`? I feel that hiding all the output here is a net-negative, especially if the container app is being created via `deploy`. Have you thought about what the long-term ideal would be?

I wonder if we could bridge the gap by simply connecting the docs here, by logging a message:
"Deploying container app through application provisioning of '<specified>' bicep module..."
"Performing application provisioning for <container app>"

of the sorts.

Reply:
Yes, the requirement does not mention any output for telling about  a provisioning operation going on.
This silentConsole sends all the output to the logs, so it is still visible for someone who enables logging, but not for the main UX experience